src/middleware/FileRestrictionsMiddleware.ts

```diff
@@ -20,20 +20,24 @@ export class FileRestrictionsMiddleware extends BaseMiddleware {
 
     public handler(req: Request, res: Response, next: NextFunction): void {
 
-        const session = req.session;
+        const session: Session | undefined = req.session;
 
-        const signInInfo: ISignInInfo =
-            session.chain(_ => _.getValue<ISignInInfo>(SessionKey.SignInInfo))
-                .ifNothing(() => loggerInstance().error(`${FileRestrictionsMiddleware.name} - Sign in info was expected in session but none found`))
-                .unsafeCoerce();
+        const signInInfo: ISignInInfo | undefined = session!.get<ISignInInfo>(SessionKey.SignInInfo);
 
-        const appeal: Appeal = session.chain(_ => _.getExtraData())
-            .chainNullable<ApplicationData>(extraData => extraData[APPLICATION_DATA_KEY])
-            .chainNullable(appData => appData.appeal)
-            .ifNothing(() => loggerInstance().error(`${FileRestrictionsMiddleware.name} - Appeal was expected in session but none found`))
-            .unsafeCoerce();
+        if (!signInInfo) {
+            loggerInstance().error(`${FileRestrictionsMiddleware.name} - Sign in info was expected in session but none found`);
```

Comment:
throw an error if there is no signInInfo, there's no point in continuing. And delete the log